ts/features/itwallet/common/components/ItwReleaserName.tsx

```diff
@@ -38,6 +39,12 @@ export const ItwReleaserName = ({ credential, isPreview }: Props) => {
       }
     ]
   });
+
+  const onPressWithMixpanelEvent = useCallback(() => {
+    trackWalletCredentialShowIssuer(credential.credential);
```

Comment:
I don't think we should send the whole raw credential to mixpanel. This is very dangerous. `ITW_CREDENTIAL_SHOW_ISSUER` is supposed to track the type of the credential.